test/utils/build.js

```diff
@@ -2,6 +2,7 @@ import { loadFixture, Nuxt, Builder } from './index'
 
 export const buildFixture = function buildFixture(fixture) {
   test(`Build ${fixture}`, async () => {
+    jest.setTimeout(120000)
```

Comment:
I think you can also add the timeout as the 3rd argument of the `test` function if you want to set it only for this specific function :)

Reply:
Good idea